### ISSUE TITLE
Global per-system configuration

### ISSUE THREAD

**Issue by JonRoma** [techservicesillinois/awscli-login#3]
Think about setting up a global configuration that is read before the per-user configuration to set system defaults (like logging configuration).

**Comment by JonRoma** [comment]
@ddriddle, when reopening an issue, would it be helpful to add a comment explaining the thinking going into that decision? Thanks.

**Comment by ddriddle** [comment]
@JonRoma yes it would. I was thinking it would be nice to have the default profile be a kind of well default for the `username`, `factor`, and `ecp_endpoint_url`. Things that would often be repeated. That would allow `.config` to be dryer.

Is that what you were interested in? Or something else?

**Comment by edthedev** [comment]
Pulling this into a new milestone view since it feels silly to keep a milestone named 'Usability Update 1' around forever. :)
